README.md

```diff
@@ -2308,8 +2294,6 @@ Bookmarks
 
 <dl>
 
-[IS651 | Distributed Systems](https://userpages.umbc.edu/~jianwu/is651/651.syll.s18.html)
```

Comment:
This would make for a suitable replacement: https://sduan.informationsystems.umbc.edu/classes/IS651/